tmt/steps/provision/podman.py

```diff
@@ -151,6 +151,37 @@ def start(self):
             ['--name', self.container, '-v', f'{workdir}:{workdir}:Z',
              '-itd', self.image])[0].strip()
 
+    def reboot(self, hard=False, container_name=None):
+        if not self.podman:
+            self.debug('container', 'No podman instance provided.', 'red')
+            return
+
+        self.info('container', 'Rebooting via podman', 'green')
+        container_name_win = container_name
+        if self.container:
+            container_name_win = self.container
+
+        if hard:
+            self.podman(['container', 'stop', container_name_win])
+            self.podman(['container', 'start', container_name_win])
+        else:
+            self.podman(['container', 'restart', container_name_win])
+
+        self.debug("Wait for an ssh connection to the machine.")
```

Comment:
Since this whole block is the same as in `GuestConnect`, it would be nice to create a method, something like `reconnect()`. Also in container we aren't really waiting for ssh connection, are we? Communication with containers is done solely through podman command so there is no ssh AFAIK

Reply:
Ok. Seems to be solved.

Reply:
This does not seems to be fixed, seems the trivial smoke test with container reveals the problem:
https://github.com/psss/tmt/pull/794#issuecomment-899600073